### ISSUE TITLE
Can't multi-cursor delete new lines

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#2684]
### Short Summary

If you do multiple cursors at the start of multiple lines you will not be able to backspace

### Steps to Reproduce

example code, `|` is a cursor:
```
|blah
|blah
|blah
```
If you setup your edit like the above example you will not be able to press Backspace and have anything happen.

Seems like it should just work.

**Comment by mitchell-as** [comment]
This has been brought up before on the Scintilla mailing list. The question is what is the expected behavior? Should all lines collapse onto one line? (e.g. you get `blahblahblah`) Or should the cursors wrap around to the previous line and continue backspacing on 3 lines? It is not clear what the best behavior is, and I don't think it's worth attempting to patch Scintilla to handle one form or another when it might change down the road. Sorry.

**Comment by cgchoffman** [comment]
Huh.  To be frank it makes no sense that there is even a question here. Why would it do anything but delete a character?  What the wrapping to line 3 about?  Are there examples where that happens?  If it was a single cursor then line 1 would do nothing.  Why would multi cursor act any differently?

**Comment by mitchell-as** [comment]
You're right. My memory was a bit flakey. This enhancement will require a Scintilla patch.

**Comment by cgchoffman** [comment]
Ok.  Thanks for clarifying in chat.  Seems like Scintilla should be handling this rather than us patching.  Think Scintilla would be open to a PR if we did it?

**Comment by mitchell-as** [comment]
Neil might be open. I send my patches upstream and most have been accepted.
